tests/crypto/crypto_hash/testcase.yaml

```
@@ -3,6 +3,7 @@ tests:
     platform_allow:
       - native_sim
       - nucleo_u575zi_q
+      - rpi_pico2/rp2350a/m33
```

nit: (not a blocker): I imagine this should just work with the Hazard3 cores as well, since there's nothing specific to the Cortex M33s in your driver.
```suggestion
      - rpi_pico2/rp2350a/hazard3
      - rpi_pico2/rp2350a/m33
```